instrumentation/servlet/servlet-3.0/javaagent-tests/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+plugins {
```

Comment:
Typically such modules are named `testing`, we currently don't have any other modules named `javaagent-tests`.
It took me a while to figure out why you did this and I was able to figure it out only because it came up on slack. I think you should add a comment about why you are doing this.

Reply:
good point